### ISSUE TITLE
Update Readme with launch options

### ISSUE THREAD

**Issue by mattsse** [ChainSafe/PINT#151]
## Issue summary

- real-overseer was dropped, instructions should include polkadot-launch:

```
git clone https://github.com/paritytech/polkadot-launch.git
cd polkadot-launch
yarn
yarn build
chmod +x dist/index.js
npm link
```

## Other information and links

-

**Comment by clearloop** [comment]
We also can provide a docker image for launching PINT like https://github.com/ChainSafe/PINT/blob/main/docker/launch.Dockerfile, or some `docker-compose.yml `

For launching PINT in terminal directly, it requires the binary of `polkadot` and `statemint`, the context of `## launch` seems will be quite verbose?

Or we provide the binaries of `polkadot` and `statemint` in the releases of our repo...

---

Since we have `js/polkadot-launch` as a submodule, we can download it directly with 

```
$ git submodule update --init
```

**Comment by mattsse** [comment]
see also https://github.com/galacticcouncil/Basilisk-node#run

**Comment by mattsse** [comment]
Closed by #172
